content/projects/tdd/string-calculator-part-1/_index.md

```diff
@@ -193,6 +186,6 @@ add("//***\n1***2***3")
 
 - Ensure that the add function takes a string as an argument.
 
-- The function that needs to be constantly updated is the `add()` function. It is even better if the `add()` function relies on other tiny functions.
+- The function that needs to be constantly updated is the `add` function. It is even better if the `add` function relies on other tiny functions.
```

Comment:
I'm not really sure what this sentence means. 

Can you also please fix the formatting of this list? I'm not sure why there are blank lines between all the list items.